examples/riemann.config

```diff
@@ -10,8 +10,14 @@
   (ws-server {:host host :port 8087})
 )
 
-(mbeans/instrumentation {:interval 10 :mbeans [{:mbean "java.lang:type=Memory" :property :HeapMemoryUsage :attribute :used}
-                                               {:mbean "java.lang:type=Runtime" :property :Uptime}]})
+(mbeans/instrumentation
+  {:interval 10
+   :mbeans [{:mbean "java.lang:type=Memory" :property :HeapMemoryUsage :attribute :used}
+            {:mbean "java.lang:type=Runtime" :property :Uptime}
+            {:mbean "java.lang:type=GarbageCollector,name=ParNew"
+             :traverse '(:LastGcInfo :memoryUsageAfterGc (keyword "Code Cache") :value :used)}
```

Comment:
Ah, I had caught that too but forgot to update the docs. Update incoming, overwriting this commit.